includes/TingSearchCqlDoctor.class.inc

```diff
@@ -118,6 +130,32 @@ class TingSearchCqlDoctor {
     return implode(' and ', $expressions);
   }
 
+
+  /**
+   * Some replacements are nested in paranthesis and/or qoutes
+   * eg. (hund and "("hest")") which is perfectly legal.
+   * Cql doctor first handles the qoutes and then the
+   * paranthesis ; thus (hund and "("hest")")  becomes encoded multiple times.
+   * This method runs through all parts to fix it
+   *
+   * @param array $expressions; The parts of the searchquery to be cured
```

Comment:
Comment should be in the form

* @param array $expressions
*    The parts of the search query to be cured.